src/js/clickable-component.js

```diff
@@ -41,9 +46,20 @@ class ClickableComponent extends Component {
       this.controlText(this.options_.controlText);
     }
 
+    const selectClickHandler = () => {
+      if (typeof this.options_.throttle === 'number' || this.options_.throttle === true) {
+        const wait = typeof this.options_.throttle === 'number' ? parseInt(this.options_.throttle, 10) : 50;
+
+        return throttle(this.handleClick.bind(this), wait);
+      }
+      return this.handleClick.bind(this);
+    };
+
+    const selectedClickHander = selectClickHandler();
+
     this.handleMouseOver_ = (e) => this.handleMouseOver(e);
     this.handleMouseOut_ = (e) => this.handleMouseOut(e);
-    this.handleClick_ = (e) => this.handleClick(e);
```

Comment:
since we still want a bound handleClick inside of selectedClickHandler, should we keep this around somewhere and use that rather than rebinding inside selectedClickHandler?